edb/server/compiler/dbstate.py

```diff
@@ -226,6 +229,12 @@ class QueryUnit:
     tx_savepoint_rollback: bool = False
     tx_savepoint_declare: bool = False
 
+    # True if this unit is one of the *explicit* transaction control commands
+    # including `START TRANSACTION`, `COMMIT`, `ROLLBACK` and the SAVEPOINT
+    # commands, excluding migration control commands `START MIGRATION ...`,
+    # `COMMIT MIGRATION` and `ABORT MIGRATION`, even though they do control tx
+    tx_control: bool = False
```

Comment:
Basically this is `tx_id is not None or tx_commit or tx_rollback or tx_savepoint_rollback or tx_savepoint_declare`? Maybe just make it a property with expression? Or, better, fold all the `tx_*` props into one Enum field.

Reply:
This was a result of my confusion among code review fixes. I didn't force-push to clear this, so it was easier to review the next fix.